topics/reference_guide/custom_language_support/websymbols_integration.md

```diff
@@ -0,0 +1,309 @@
+<!-- Copyright 2000-2023 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# Web Symbols Integration with language features
+
+<link-summary>
+How to integrate your Web Symbols with a particular language feature.
+</link-summary>
+
+IDEs provide built-in support for Web Symbols integration with main language features of HTML, CSS and JavaScript.
+You can contribute static symbols through Web Types or Custom Elements Manifest, or you can
+create and register [`WebSymbolsQueryConfigurator`](%gh-ic%/platform/webSymbols/src/com/intellij/webSymbols/query/WebSymbolsQueryConfigurator.kt)
+extension.
+
+Your implementation of `WebSymbolsQueryConfigurator` needs to override `getScope` method, in which you
+create a list of `WebSymbolScope`s depending on the provided `PsiElement` (`element` parameter) and `WebSymbolContext` (`context` parameter).
+The below list of supported language features specify information on what kind of `PsiElement`s to expect for each of
+the supported language features.
+
+## Supported language features
+
+### HTML
+
+#### Elements {#html-elements}
+
+|---|---|
```

Comment:
we shouldn't use a table for two pieces of information

suggestion:

```
**Namespace**: `html`

**Kind**: `elements`
```

or similar, apply to all on this page

Reply:
Hmm, the table looks much nicer graphically, I've experimented with different approaches, including your suggestion, but table looked the best for me. Have you experimented and you're sure your suggestion looks the best for you?